app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+	  
+  protect_from_forgery with: :null_session
+  before_action :authenticate_user!
+
+  def list
+    @users = User.all
+    # Add tabs for user types
+    #@users.each do |m|
+    #  m.is_done ? @done+=1 : @not_done+=1
+    #end
```

Comment:
Czemu to zostało wykomentowane? Do usunięcia, czy będzie potrzebne?

Reply:
W sumie po nic, kiedyś chce zrobić zakładki ale w tym momencie ani to potrzebne ani wymagane.

Usunięte w c29b8fdba679dd974086063740f4eb2305f53ccd